README.md

```diff
@@ -1,5 +1,11 @@
 # Kn
 
+[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/knative.dev/client)
+[![Go Report Card](https://goreportcard.com/badge/knative/serving)](https://goreportcard.com/report/knative/client)
```

Comment:
Why is there a link for serving and client? Should it just be client?

Reply:
fixed